### ISSUE TITLE
Hello, what about iOS version ?

### ISSUE THREAD

**Issue by numerized** [tijmenvangulik/cordova-usb-hid#4]
Hello brother, thanks for the efforts you put here.
Because of your work I could make work my app after Android update that killed the previous plugin ...

But I was wondering if the same is possible with iOS, I can't seem to find any repo with the iOS HID feature.

Do you have any idea about that feasibility ?

**Comment by tijmenvangulik** [comment]
No I also can not find this, but it should be possible as concept2 provides a usb link for ergdata running on ios and the pm5 is a hid device


https://www.concept2.nl/nl/service/software/ergdata/gebruiksaanwijzing/het-ios-apparaat-aansluiten-op-de-indoor-rower￼

**Comment by numerized** [comment]
okay, so it's not impossible, I thought was possible but on iOS the device has to be registered officially in apple systems.
I read that some time ago. It's a kind of licence to make that work.

**Comment by tijmenvangulik** [comment]
yes I read some thing the same in the past when I researched this.

Tijmen
